3rdPartyCypress/cypress/support/commands.js

```diff
@@ -3,7 +3,6 @@ Cypress.Commands.add('testResponseCodeAndType', (path, responseCode, type) => {
   cy.request({
     url: path,
     failOnStatusCode: false,
-    retryOnStatusCodeFailure: true,
```

Comment:
Any reason why this was removed?

Reply:
It didn't work! `failOnStatusCode: false` and `retryOnStatusCodeFailure: true` are mutually exclusive 😄